lib/util/source-code-fixer.js

```diff
@@ -70,6 +70,15 @@ SourceCodeFixer.applyFixes = function(sourceCode, messages) {
         };
     }
 
+    if (shouldFix === false) {
+        debug("shouldFix parameter was false, not attempting fixes");
+        return {
+            fixed: false,
+            messages,
+            output: ""
```

Comment:
Should the output here be `sourceCode.text` rather than the empty string, since the resulting text is the same as the original text? (I'm not sure whether the other parts of the code check the `fixed` value before using the output, but setting the output to the original text seems like it would be least surprising in case `fixed` isn't checked.)